src/app/pages/administration/search/landing-page.jsx

```diff
@@ -134,15 +137,15 @@ function ChplLandingPage() {
       <Box className={classes.landingPageBackground} pb={16} height="fit-content">
         <Box className={classes.landingPageImageryBackground}>
           <Container maxWidth="md">
-            <Box className={classes.searchContainer} mt={-8}>
+            <Box className={classes.searchContainer}>
               <ChplFilterSearchTerm />
-              <ChplFilterPanel />
-              <ChplFilterQuickFilters />
+              <Box display="flex" justifyContent="space-between">
```

Comment:
Why aren't these elements in a class? I know `<Box>`s allow putting stuff in here, but how is that different from the old `<div style="color: red; border: 1 px sold">` or something?